controllers/suite_test.go

```diff
@@ -172,89 +166,3 @@ var _ = AfterSuite(func() {
 	By("tearing down the test environment")
 	testEnv.Stop()
 })
-
-func mainFunction(scheme *runtime.Scheme) (manager.Manager, error) {
```

Comment:
No need for main anymore?

Reply:
It is currently being unused. @vivekr-splunk Do we need to keep it?

Reply:
Spoke with @vivekr-splunk , we can remove for now and revisit later if required.